conf/riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks: [membership]
+regions: [eu-west-1]
+deployments:
+  cfn:
+    type: cloud-formation
+    app: contributions-frontend
+    parameters:
+      prependStackToCloudFormationStackName: false
```

Comment:
I believe it's already searching by tags as `true` is the default value for `cloudFormationStackByTags`

I kept `prependStackToCloudFormationStackName` just in case we would destroy the stack and let riff-raff re-create it, to keep a similar name